tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -225,15 +230,111 @@ def enroll_idm_and_configure_external_auth():
         cmd='ipa service-add HTTP/{}'.format(settings.server.hostname),
         hostname=settings.ipa.hostname_ipa,
     )
-    _desired_pos = settings.ipa.hostname_ipa.find('satqe')
-    domain = settings.ipa.hostname_ipa[_desired_pos:]
+    domain = settings.ipa.hostname_ipa[settings.ipa.hostname_ipa.find('satqe') :]  # noqa
```

Comment:
This is better (shorter, without `# noqa`) written with split (at least as long as FQDN here doesn't change too much):
```python
_, domain = settings.ipa.hostname_ipa.split('.', 1)
```
Proof:
```python
>>> _, domain = settings.ipa.hostname_ipa.split('.', 1)
>>> domain == settings.ipa.hostname_ipa[settings.ipa.hostname_ipa.find('satqe') :]
True
```